packages/editor/src/plugins/serlo-template-plugins/course/course.tsx

```diff
@@ -152,14 +136,75 @@ function CourseTypeEditor(props: EditorPluginProps<CourseTypePluginState>) {
             onChange={(e) => title.set(e.target.value)}
           />
         }
-        pages={staticPages.map((coursePage) => {
+        pages={staticPages.map(({ title, id }, index) => {
           return {
-            title: coursePage.title,
-            url: `#page-${coursePage.id}`,
-            id: coursePage.id,
+            key: title + id + index,
+            element: (
+              <div className="group">
+                <button
+                  onClick={() => {
+                    if (index === activePageIndex) return
+                    setActivePageIndex(index)
+                    window.location.hash = `#${staticPages[activePageIndex].id}`
+                  }}
+                  className={cn(
+                    'serlo-link text-lg leading-browser',
+                    index === activePageIndex &&
+                      'font-semibold text-almost-black hover:no-underline'
+                  )}
+                >
+                  {title.trim().length ? title : '___'}
+                </button>{' '}
+                {/* {index > 0 ? (
```

Comment:
Can we remove this comment block?

Reply:
added a comment, usually I'd remove unused code, but this will hopefully become relevant soonish…